USF-1670: Dropins Localization (#95)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/user-account/account-dictionary.mdx

---

> title: User Account Dictionary

Reviewer: ```suggestion
title: User account dictionary
```

> description: Reference the default dictionary for the User Account dropin.

Reviewer: ```suggestion
description: Learn how to customize the default dictionary for the user account drop-in component.
```

---

import CodeImport from '@components/CodeImport.astro';
import CodeInclude from '@components/CodeInclude.astro';
import dictionary from '@dropins/storefront-account/i18n/en_US.json.d.ts?raw';

> The default User Account dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the User Account drop-in.

Reviewer: ```suggestion
The default user account dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the user account drop-in component.
```
